Cargo.toml

```diff
@@ -6,7 +6,7 @@ documentation = "https://docs.rs/bellperson"
 homepage = "https://github.com/filecoin-project/bellman"
 license = "MIT/Apache-2.0"
 repository = "https://github.com/filecoin-project/bellman"
-version = "0.9.0"
+version = "0.8.0"
```

Comment:
probably don't change the vesion in a PR ;)

Reply:
That is a temporary inclusion in the branch to allow downstream patching. Will remove before merging. I mentioned elsewhere but apparently not in the description here. I agree.